server/form-pages/apply/reasons-for-placement/type-of-ap/apType.ts

```diff
@@ -52,27 +52,25 @@ export default class SelectApType implements TasklistPage {
   items() {
     return convertKeyValuePairToRadioItems(apTypeLabels, this.body.type, undefined, {
       mhapElliottHouse: {
-        html: `
-          <p class="govuk-body govuk-hint">To apply for this type of placement a person must:</p>
-          <ul class="govuk-list govuk-list--bullet govuk-hint">
-            <li>have a confirmed diagnosis of a severe and enduring mental illness e.g paranoid schizophrenia / bipolar affective disorder</li>
-            <li>be managed by the Probation Service in Greater Manchester</li>
-          </ul>
-          <p class="govuk-body govuk-hint">They must also:</p>
-          <ul class="govuk-list govuk-list--bullet govuk-hint">
-            <li>have had a total of 6 months in a psychiatric ward or day hospital; or</li>
-            <li>have had 3 admissions to hospital or day hospital; or</li>
-            <li>have had 6 months of psychiatric community care involving more than one worker or the perceived need for such care if unavailable or refused; or</li>
-            <li>be under the care of community mental health services and the CPA process; or</li>
-            <li>be under the care of secondary care mental health services within a custodial setting as an alternative to community mental health services</li>
-          </ul>
-          <p class="govuk-body govuk-hint">You will be required to submit a <a class="govuk-link" href="https://justiceuk.sharepoint.com/:w:/s/ApprovedPremisesCareerDevelopment/Eag6XVxypVBHhu-pBMrfvqIBTTva2uNFkKB9Tl6c6OX-jA?e=sGaEma">supplementary form (opens in a new tab)</a>.Once completed it should be uploaded as part of the 'Add documents' section of this application before submission.</p>
+        html: `<p class="govuk-body govuk-hint">To apply for this type of placement a person must have a confirmed diagnosis of a severe and enduring mental illness e.g. paranoid schizophrenia / bipolar affective disorder (the primary diagnosis should not be personality disorder).</p>
```

Comment:
I wonder if this html can be stored elsewhere? It makes the code look a bit busy